tlscommon.c

```diff
@@ -626,13 +626,38 @@ static int certattr_matchip(GENERAL_NAME *gn, struct certattrmatch *match){
         && !memcmp(ASN1_STRING_get0_data(gn->d.iPAddress), &match->ipaddr, l)) ? 1 : 0 ;
 }
 
+static int compareWithModifiedHostname(char* certDNS, char* scriptHostName) {
+    int len = 0;
+    int len1 = 0, len2 = 0;
```

Comment:
@KarthikaRajendran11 , these variables' names could be more meaningful. The current ones make it hard to follow the logic.